logic.js

```diff
@@ -198,6 +208,28 @@ function load_ability_deck(deck_definition)
       }
     }
 
+    deck.set_real_name = function(real_name)
+    {
+        // This will serve to know when we can load the monster data (Move/Attack)
+        this.real_name = real_name;
+        this.draw_pile.concat(this.discard).forEach(
+            function(card) {
+                card.change_displaying_name(real_name);
+            });
+    }
+
+    deck.clean_real_name = function()
+    {
+      if (this.real_name)

```

Comment:
Hate my computers. One of them has the indentation to 2 spaces... :( And can't tell which is the one!